pkgs/tools/networking/shadowsocks-libev/default.nix

```diff
@@ -24,6 +24,19 @@ stdenv.mkDerivation rec {
     cp lib/* $out/lib
     chmod +x $out/bin/*
     mv $out/pkgconfig $out/lib
+
+    ${stdenv.lib.optionalString stdenv.isDarwin ''
+      install_name_tool -change libcork.dylib $out/lib/libcork.dylib $out/lib/libipset.dylib
+      install_name_tool -change libbloom.dylib $out/lib/libbloom.dylib $out/lib/libipset.dylib
+
+      for exe in $out/bin/*; do
+        install_name_tool -change libmbedtls.dylib ${mbedtls}/lib/libmbedtls.dylib $exe
```

Comment:
This only fixes the current library references, not the install_name (see. `otool -D` and `install_name_tools -id`). This means the binaries now work but anything that links against the libraries will run into the same issues.

Reply:
should I revert?

Reply:
It's ok, the package itself will work properly now and there's currently nothing in nixpkgs that links against it.